Protos/unittest_swift_naming.proto

```diff
@@ -720,7 +720,12 @@ enum EnumFieldNames {
     __FUNCTION__ = 63;
     __LINE__ = 64;
     _ = 65;
-    __ = 1065;
```

Comment:
I'll make a second.

Just want to confirm, nothing uses this enum, so there isn't anything else I should update as part of adding the second enum?